### PR TITLE
dashboards: initial support for tables in monitoring dashboards

### PR DESCRIPTION
**Pull request by spadgett**
/cc @rhamilto @benjaminapetersen @kyoto 
/kind feature

Screen Shot 2020-01-03 at 2 42 17 PM

### CODE REVIEW

**Comment by spadgett** [comment]
The tables take up more space than in Grafana in part due to the PF styles. We should look at what we can do there.

Not all of the numbers are normalized yet (like pps).

We should add sorting to the table column headers.

There's some other table capabilities like drilldown we haven't looked at.

File: frontend/public/components/monitoring/dashboards/table.tsx [hunk 1]
```diff
@@ -1,10 +1,154 @@
...
+  formatToFractionalDigits,
+  humanizeBinaryBytes,
+  humanizeDecimalBytesPerSec,
+  LoadingInline,
+  usePoll,
+  useSafeFetch,
+} from '../../utils';
+import { TablePagination } from '../metrics';
+
+const formatNumber = (value: number, decimals: number, unit: string): string => {
```
Comment (spadgett) [inline frontend/public/components/monitoring/dashboards/table.tsx]: @kyoto we can make this resuable for other cards like `SingleStat`

**Comment by spadgett** [comment]
There's at least one table that doesn't use `transform: table`, the "Cluster Status" table on the `cluster-total` dashboard.

**Comment by rhamilto** [comment]
/hold

probably should squash

**Comment by spadgett** [comment]
squashed

**Comment by spadgett** [comment]
/hold
the merge queue is blocked

**Pull request merged**